frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-tag-input/common-tag-input.component.ts

```diff
@@ -18,6 +18,17 @@ import { TranslateModule } from '@ngx-translate/core';
 // Example Usage:
 //   <app-common-tags-input formControlName="tags"></app-common-tags-input>
 
+// To add Import/Export button while using component
+// you can add the property 'showUploadIcon' or 'showDownloadIcon'
+// with its repective binding 'uploadClicked' or 'downloadClicked'
```

Comment:
The tsdoc looks like it needs some updates, the wording is a little different and the usage should show import also. Could even do two examples to demonstrate that the typical usage is simpler than when dealing with buttons:

```
// Typical usage
// <app-common-tags-input
//         formControlName="tags"
//       ></app-common-tags-input>
//
// Manage built-in optional action buttons
// <app-common-tags-input
//         [showExportIcon]="true"
//         (exportClicked)="onExportClick()"
//         [showExportIcon]="false"
//         (importClicked)="onImportClick()"
//         formControlName="tags"
//       ></app-common-tags-input>
```